powersimdata/network/constants/storage.py

```diff
@@ -0,0 +1,21 @@
+storage = {
+    "duration": 4,
+    "min_stor": 0.05,
+    "max_stor": 0.95,
+    "InEff": 0.9,
+    "OutEff": 0.9,
+    "energy_value": 20,
+    "LossFactor": 0,
+    "terminal_min": 0,
+    "terminal_max": 1,
+}
+
+
+def get_storage(model):
+    """Return storage constants.
+
+    :param str model: grid model
+    :return: (*dict*) -- storage information.
+    """
+    if model in ["usa_tamu", "hifld"]:
+        return storage
```

Comment:
Can we check against another location that has these entries, like `Grid.SUPPORTED_MODELS` or `powersimdata.network.constants.model.model2region`, or would that cause a circular import?

Should we raise an Exception if `model` is an invalid input, rather than silently returning `None`?

Reply:
Thanks. It should be cleaner now.